ui/ui_v1/scripts/code.js

```diff
@@ -0,0 +1,68 @@
+
+//onOpen trigger runs automatically when a user opens a spreadsheet
+function onOpen() {
+  var ui = SpreadsheetApp.getUi();
+  ui.createAddonMenu()
+    .addItem('Show','show')
+    .addItem('Top-k','topK')
+    .addItem('Trend','trend')
+    .addSeparator()
+    .addItem('Slice-Compare','sliceCompare')
+    .addItem('Time-Compare','timeCompare')
+    .addSeparator()
+    .addItem('Growth','growth')
+    .addToUi();
+}
+
+//onInstall trigger runs automatically when a user installs add-on
+function onInstall() {
+  ui.alert('Installed');
+  onOpen();
+}
+
+//function invoked on clicking show from add-on menu bar
+function show() {
+  var html= HtmlService
+              .createTemplateFromFile('html/intents/show')
+              .evaluate()
+              .setTitle('Show');
+  SpreadsheetApp.getUi().showSidebar(html);
+}
+
+//function invoked on clicking top-k from add-on menu bar
+function topK() {
+  var html= HtmlService
+              .createTemplateFromFile('html/intents/topk')
+              .evaluate()
+              .setTitle('Top-K');
+  SpreadsheetApp.getUi().showSidebar(html);
+}
+
+//function to import css and javascript files into the html file
+function include(filename) {
+  return HtmlService.createHtmlOutputFromFile(filename).getContent();
+}
+
+//function to select the activated range from sheet
+function getSelectedRange() {
+  var range = SpreadsheetApp.getActiveSheet().getActiveRange(); 
+  var rangeString = range.getA1Notation(); 
+  var headerRow = range.getLastRow() - range.getHeight()+1;
```

Comment:
This is implying that header row is always the row above the selection right ? If yes can you mention it here.

Reply:
It implies that header row will be the first row of the selected range. I will add this in comments here.